src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -923,6 +923,45 @@ public Integer getNumberOfCorrectionRounds() {
         }
     }
 
+    /** Helper method which does a hard copy of the Grading Criteria
+     *
+     *
+     * @param exerciseImportService@return A clone of the grading criteria list
```

Comment:
Nitpicky, but as @lschlesinger mentioned a new line should be added before the return and I would only add one new line after the initial description.

```suggestion
    /** Helper method which does a hard copy of the Grading Criteria
     *
     * @param exerciseImportService
     * @return A clone of the grading criteria list
```